README.md

```diff
@@ -30,8 +30,21 @@ Once you updated the code, you can build it with:
 npm run build
 ```
 
-And serve the build with:
+And server the build with:
```

Comment:
I think it's a typo, it should be `serve` as it was before

Reply:
modified.